crypto/src/main/java/com/strategyobject/substrateclient/crypto/Hasher.java

```diff
@@ -16,7 +16,8 @@ public static byte[] blake2(Size size, byte[] value) {
     }
 
     public static long xx(long seed, byte[] value) {
-        return LongHashFunction.xx(seed).hashBytes(value);
+        val f = LongHashFunction.xx(seed);
```

Comment:
What is a purpose of that?

Reply:
it is an artifact. I noticed an issue in LongHashFunction and took a look into it.